src/main/java/com/kodilla/ecommercee/mapper/OrderMapper.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.OrderDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderMapper {
+    public Order mapToOrder(final OrderDto orderDto) {
+        return new Order(
+                orderDto.getId(),
```

Comment:
z tym getId chyba będzie ten sam problem, co u Pawła - nie powinno go tu być

Reply:
ale to jest mapowanie obiektów. Chyba muszę wszystkie pola skopiować. W jaki sposób Ty to robisz?

Reply:
Ok, wydaje mi się, że u Pawła był problem z tym, że on tworzy Product przez podanie ProductDto a na tym etapie nie ma on przydzielonego id. Ja natomiast tworzę Order na podstawie cardId i userId i u mnie tego problemu nie ma.

Reply:
To co było u Pawła już wyjaśniłem na chat i jednak było ok :P tu też jest ok